src/pfe/file-watcher/server/src/projects/LibertyValidator.ts

```diff
@@ -143,10 +143,12 @@ export class LibertyValidator extends Validator {
                                 let libertyPluginFound = false;
                                 try {
                                     if (mcprofile.build[0].plugins[0].plugin) {
-                                        mcprofile.build[0].plugins[0].plugin.forEach(async (_plugin: any) => {
+                                         for ( const _plugin of  mcprofile.build[0].plugins[0].plugin ) {
+                                            logger.logInfo("_plugin.groupId: " + _plugin.groupId[0]);
```

Comment:
think we should have these logs as logTrace or logDebug

Reply:
That's the log message I used for debugging purpose. Should be removed. Thanks for catching it.